CHANGELOG.md

```diff
@@ -1,5 +1,114 @@
+
+- Add char range overload to the `..` operator, so `'a'..'z'` returns characters from 'a' to 'z'. (#5026) (Thanks @IISResetMe!)
+- Remove `CommandFactory` because it serves no real purpose. (#5266)
+- Change to not insert line breaks at console window width to output (except for tables). (#5193)
+- Use `Ast` for context in parameter binding and fix to glob the native command argument only when it's not quoted. (#5188)
+- Fix dynamic class assembly name. (#5292)
+- Update powershell to use `2.0.4-servicing` dotnet core runtime. (#5295)
+- Fix `ExecutionContext.LoadAssembly` to load with name when file cannot be found. (#5161)
+- Speed up the check for suspicious content in script texts. (#5302)
+- Use native `os_log` APIs on macOS for PowerShell Core logging. (#5310)
+- Redirect `ETW` logging to `Syslog` on Linux. (#5144)
+- Improve how we pass the array literal to native commands. (#5301)
+- Make `SemanticVersion` compatible with `SemVer 2.0`. (#5037) (Thanks @iSazonov!)
+- Revert refactoring changes that broke remoting to Windows PowerShell 5.1. (#5321)
+- Port some fixes in `Job` for an issue that causes PowerShell to not respond. (#5258)
+- Multiple improvements by `CodeRush` static analysis. (#5132) (Thanks @Himura2la!)
+- Fix the Runspace cleanup issue that causes PowerShell to not respond on exit. (#5356)
+- Update PowerShell to depend on new version of `psrp` and `libmi` nuget packages on Unix platforms. (#5469)
+
+### General cmdlet updates and fixes
+
+- Add `-AsHashtable` to `ConvertFrom-Json` to return a `Hashtable` instead. (#5043) (Thanks @bergmeister!)
+- Fix `Import-module` to not report a loaded module was not found. (#5238)
+- Fix performance issues in `Add-Type`. (#5243) (Thanks @iSazonov!)
+- Fix `PSUserAgent` generation for Web Cmdlets on Windows 7. (#5256) (Thanks @markekraus!)
+- Remove `DCOM` support from `*-Computer` cmdlets. (#5277)
+- Add multiple link header support to Web Cmdlets. (#5265) (Thanks @markekraus!)
+- Use wider columns for process id and user. (#5303)
+- Add `Remove-Alias` Command. (#5143) (Thanks @PowershellNinja!)
+- Update `installpsh-suse.sh` to work with the `tar.gz` package. (#5309)
+- Add `Jobject` serialization support to `ConvertTo-Json`. (#5141)
+- Display full help with 'help' function. (#5195) (Thanks @rkeithhill!)
+- Fix `help` function to not pipe to `more` if objects are returned instead of help text. (#5395)
+- Fix `Unblock-File` to not write an error if the file is already unblocked. (#5362) (Thanks @iSazonov!)
+- Clean up FullCLR code from Web Cmdlets. (#5376) (Thanks @markekraus!)
+- Exclude cmdlets that are not supported on Unix platforms. (#5083)
+- Make `Import-Csv` support `CR`, `LF` and `CRLF` as line delimiters. (#5363) (Thanks @iSazonov!)
+- Fix spelling in Web Cmdlet errors. (#5427) (Thanks @markekraus!)
+- Add `SslProtocol` support to Web Cmdlets. (#5329) (Thanks @markekraus!)
+- Add `PSTypeName` Support for `Import-Csv` and `ConvertFrom-Csv`. (#5389) (Thanks @markekraus!)
```

Comment:
Are we considering this one a breaking change? technically it is.. but... thew likelihood it actually affects anyone is pretty low.

Reply:
It turns out the issue #5134 is marked with 'Breaking-Change'. I will move this one to the breaking change section.